### PR TITLE
Add gbsyncd container for broncos

### PR DESCRIPTION
**Pull request by andywongarista**
#### Why I did it
This change is to support platforms that use the Broncos (BCM54182) external PHY chips.
#### How I did it
The change introduces a new docker-gbsyncd-broncos container, similar to the container for Credo added in https://github.com/Azure/sonic-buildimage/pull/8144
The gbsyncd.ini file is added for determining which gbsyncd container a platform needs.
By default, the container will not be included in the built image unless the (LIBSAI_BRONCOS)_URL is defined (pointing to a deb package containing the broncos driver).  

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [x] 202205

### CODE REVIEW

File: platform/broadcom/one-aboot.mk [hunk 1]
```diff
@@ -4,7 +4,7 @@ SONIC_ONE_ABOOT_IMAGE = sonic-aboot-broadcom.swi
 $(SONIC_ONE_ABOOT_IMAGE)_MACHINE = broadcom
 $(SONIC_ONE_ABOOT_IMAGE)_DEPENDENT_MACHINE = broadcom-dnx
 $(SONIC_ONE_ABOOT_IMAGE)_IMAGE_TYPE = aboot
-$(SONIC_ONE_ABOOT_IMAGE)_INSTALLS += $(PHY_CREDO)
+$(SONIC_ONE_ABOOT_IMAGE)_INSTALLS += $(PHY_CREDO) $(PHY_BRONCOS)
```
Comment (jimmyzhai) [inline platform/broadcom/one-aboot.mk:7]: Not find where `PHY_BRONCOS` is defined

File: platform/broadcom/rules.dep [hunk 2]
```diff
@@ -30,3 +30,4 @@ include $(PLATFORM_PATH)/libsaithrift-dev.dep
 include $(PLATFORM_PATH)/docker-syncd-brcm-dnx.dep
 include $(PLATFORM_PATH)/docker-syncd-brcm-dnx-rpc.dep
 include $(PLATFORM_PATH)/../components/docker-gbsyncd-credo.dep
+include $(PLATFORM_PATH)/../components/docker-gbsyncd-broncos.dep
```
Comment (jimmyzhai) [inline platform/broadcom/rules.dep:33]: It is not desirable to include and build all gbsyncd dockers by default, with more phy chips support. Could we optionally include it with a config variable like `INCLUDE_GBSYNCD_BRONCOS`? For example:

```
ifeq ($(INCLUDE_GBSYNCD_BRONCOS), y)
include $(PLATFORM_PATH)/../components/docker-gbsyncd-broncos.dep
endif
```

File: platform/broadcom/rules.mk [hunk 3]
```diff
@@ -29,6 +29,7 @@ include $(PLATFORM_PATH)/libsaithrift-dev.mk
 include $(PLATFORM_PATH)/docker-syncd-brcm-dnx.mk
 include $(PLATFORM_PATH)/docker-syncd-brcm-dnx-rpc.mk
 include $(PLATFORM_PATH)/../components/docker-gbsyncd-credo.mk
+include $(PLATFORM_PATH)/../components/docker-gbsyncd-broncos.mk
```
Comment (jimmyzhai) [inline platform/broadcom/rules.mk:32]: As above

**Review by lguohan — APPROVED** [review]

File: platform/components/docker-gbsyncd-broncos.mk [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+ifneq ($($(LIBSAI_BRONCOS)_URL),)
+
+DOCKER_GBSYNCD_BRONCOS_STEM = docker-gbsyncd-broncos
+DOCKER_GBSYNCD_BRONCOS = $(DOCKER_GBSYNCD_BRONCOS_STEM).gz
+DOCKER_GBSYNCD_BRONCOS_DBG = $(DOCKER_GBSYNCD_BRONCOS_STEM)-$(DBG_IMAGE_MARK).gz
+
+$(DOCKER_GBSYNCD_BRONCOS)_FILES += $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
+
+$(DOCKER_GBSYNCD_BRONCOS)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE_BUSTER)
```
Comment (jimmyzhai) [inline platform/components/docker-gbsyncd-broncos.mk:12]: Please update base docker from buster to bullseye like https://github.com/Azure/sonic-buildimage/pull/11288.

File: platform/components/docker-gbsyncd-broncos/Dockerfile.j2 [hunk 5]
```diff
@@ -0,0 +1,40 @@
+FROM docker-config-engine-buster
```
Comment (jimmyzhai) [inline platform/components/docker-gbsyncd-broncos/Dockerfile.j2:1]: Please update base docker from buster to bullseye like https://github.com/Azure/sonic-buildimage/pull/11288.

**Review by jimmyzhai — APPROVED** [review]

**Pull request merged**